resources/js/forms_editor/components/EditorHeader.vue

```diff
@@ -52,6 +72,30 @@ export default {
     preview_url() {
       const form_id = this.$store.state.editor.form.id
       return `/home_staff/applications/preview/${form_id}`
+    },
+    is_public() {
+      return this.$store.state.editor.form.is_public
+    }
+  },
+  methods: {
+    save() {
+      this.$store.dispatch(`editor/${SAVE_FORM}`)
+    },
+    togglePublic() {
+      if (
+        window.confirm(
+          this.$store.state.editor.form.is_public
+            ? '非公開にしますか？'
+            : '公開しますか？\n公開しても受付期間まで回答することはできません。'
```

Comment:
確認メッセージいいね！

`公開しても受付期間まで回答することはできません。` なんだけど、受付期間終了後も回答することはできないというのと、主語を明確にしたいので、以下のようにしてもらっても良いでしょうか？

```suggestion
            : '公開しますか？\n公開しても受付期間外の場合、団体は回答できません。'
```